### PR TITLE
Update changelog for upcoming 0.27.0

### PR DESCRIPTION
**Pull request by evgenykochetkov**
### Features and enhancements

* [c++, ide] Add ability to simulate a program without uploading it to a board (#1661)
* [nodes, c++, ide] Add `xod/debug/tweak-*` nodes whose values can be adjusted right from the IDE during debug and simulation sessions (#1662, #1665, #1666)
* [ide] Add ability to connect to a board by serial at any time (#1660)
* [nodes] Move exising debug-related nodes (`watch` and `console-log`) from `xod/core` to `xod/debug` library (#1657)
* [ide] Add tooltips for show/hide sidebar pane buttons (#1658)
* [ide] Make a fancy look for `xod/patch-nodes/jumper` (#1659)

### Bug fixes

* [nodes] Fix prescision and overflow issues with `format-number` and `cast-to-string(number)` nodes (#1650, #1652, #1669)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,21 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## Not yet released
+
+### Features and enhancements
+
+* [c++, ide] Add ability to simulate a program without uploading it to a board (#1661)
```
Comment (nkrkv) [inline CHANGELOG.md]:
```suggestion
* [ide] Add ability to simulate a program without uploading it to a board (#1661)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,21 @@
...
+* [nodes, c++, ide] Add `xod/debug/tweak-*` nodes whose values can be adjusted right from the IDE during debug and simulation sessions (#1662, #1665, #1666)
```
Comment (nkrkv) [inline CHANGELOG.md]:
```suggestion
* [nodes] Add `xod/debug/tweak-*` nodes whose values can be adjusted right from the IDE during debug and simulation sessions (#1662, #1665, #1666)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,21 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## Not yet released
+
+### Features and enhancements
+
+* [ide] Add ability to simulate a program without uploading it to a board (#1661)
+* [nodes] Add `xod/debug/tweak-*` nodes whose values can be adjusted right from the IDE during debug and simulation sessions (#1662, #1665, #1666)
+* [ide] Add ability to connect to a board by serial at any time (#1660)
```
Comment (nkrkv) [inline CHANGELOG.md]:
```suggestion
* [ide] Add ability to connect to a board by serial, send text lines, and see its response at any time (#1660)
```

**Comment by evgenykochetkov** [comment]
Added #1589.

#1599 is mentioned in release notes for 0.26.0
#1655 is very minor, not sure it's worth mentioning at all
#1648 by itself means nothing for the end user

**Pull request merged**